firmware/library/L1_Drivers/LCD_I2C_Backpack/I2CBackpack.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <cstdint>
+
+// Import SJOne libraries to be used for testing
+#include "L2_Drivers/base/i2c_base.hpp"
+
+class BackpackInterface
+{
+ public:
+    enum FontSize
+    {
+        small;
```

Comment:
The rest of the naming issues will appear once I merge the new presubmit checks.